File: movie-catalog-service/src/main/java/dk/amfibia/moviecatalogservice/resources/HelloResource.java
package dk.amfibia.moviecatalogservice.resources;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/hello")
public class HelloResource {

    @RequestMapping("/who")
    public String sayHello() {
        return "Hello World";
    }

}

File: residential-catalog-service/src/main/kotlin/dk/amfibia/residencialcatalogservice/resource/HelloResource.kt
package dk.amfibia.residencialcatalogservice.resource

import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/")
class HelloResource {

    @RequestMapping("hello")
    fun hello(): String { return "Hello"}

}
